pkg/oc/admin/prune/imageprune/prune.go

```diff
@@ -578,7 +578,7 @@ func (p *pruner) addDeploymentsToGraph(dmnts *kapisext.DeploymentList) []error {
 		d := &dmnts.Items[i]
 		ref := getRef(d)
 		glog.V(4).Infof("Examining %s", getKindName(ref))
-		dNode := appsgraph.EnsureDeploymentNode(p.g, d)
+		dNode := kubegraph.EnsureDeploymentNode(p.g, d)
```

Comment:
if you wonder why, I moved 'Deployments' from appsgraph which seems to hold only `apps.openshift.io` resources to kubegraph as Deployments are upstream.

Reply:
Would you mind moving DaemonSets as well, it's the function before this one?

Reply:
Same for ReplicaSets a few lines below.

Reply:
interesting, I thought I moved replica sets already... seems like they are not used/loaded anywhere

Reply:
done. moved both to kube nodes